com.maplerain.telsis_translator_flutter.yml

```diff
@@ -0,0 +1,34 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+
+---
+app-id: com.maplerain.telsis_translator_flutter
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: telsis_translator_flutter
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+modules:
+  # telsis_translator_flutter
+  - name: telsis_translator_flutter
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - "./build-flatpak.sh"
+    sources:
+      - type: file
+        url: https://github.com/vivian-ng/telsis_translator_flutter/releases/download/0.2.1/telsis-translator-flutter_0.2.1.tar.gz
+        sha256: 118a8ff29226a041cbacceb71bb95a21c8d1ba90c020a5ee1ef3ec8c9f4f0746
+      - type: dir
```

Comment:
this is incorrect. There is no such directory.

Reply:
or more like it's `.` which in the end goes back to what I as saying. it's incorrect.

Reply:
I have removed the dir section.